e2e/tests/dump-restore-test.rs

```diff
@@ -140,13 +230,21 @@ fn snapshot_db(container: &Container<Cli, GenericImage>, db: &str, username: &st
         exit
     );
     let src = PathBuf::from("/tmp/snapshot.txt");
-    let dest = env::temp_dir().join(format!("snapshot-{}.txt", db));
-    copy_out(container, &src, &dest);
-    println!("database snapshot at: {}", dest.display());
-    fs::read_to_string(dest.as_path()).unwrap()
+    copy_out(container, &src, path);
```

Comment:
Remove the snapshot file before returning?

Reply:
I'm not deleting the files on purpose for the sake of debugging. You can open the two snapshot files in a proper diff tool if the test fails, which is super helpful